.changeset/tiny-weeks-chew.md

```diff
@@ -0,0 +1,5 @@
+---
+'@kadena/react-ui': minor
+---
+
+adding new text area component and the field for it
```

Comment:
These end up being the release notes, so they should be written as if other developers are reading it. I would suggest something like this 
```suggestion
Added the TextArea and TextAreaField components
```